### PR TITLE
Guarin lig 653 add example to docs and cli help on how

### PR DESCRIPTION
**Pull request by guarin**
* Add lightly-magic commands to cli help
* Improve docs on how to upload custom metadata

[Command-line tool — lightly 1.2.10 documentation.pdf](https://github.com/lightly-ai/lightly/files/8214632/Command-line.tool.lightly.1.2.10.documentation.pdf)

### CODE REVIEW

File: docs/source/getting_started/command_line_tool.rst [hunk 1]
```diff
@@ -96,14 +96,14 @@ Lightly-magic is a singular command for training, embedding, and uploading to th
     
 .. code-block:: bash
     
-    # embedding and uploading the images and embeddings to web app
-    # passing in input directory of dataset(Data_dir),using pretrained model
-    # with assigned token as yourToken and datasedID as yourDatasedID
-    # loader.num_workers species the number of cpu cores used for speeding the process.
+    # Embedding and uploading the images and embeddings to web app passing in 
+    # input directory of dataset(Data_dir), using pretrained model with assigned 
+    # token as yourToken and datasedID as yourDatasedID loader.num_workers specifies
```
Comment (MalteEbner) [inline docs/source/getting_started/command_line_tool.rst:101]: There was a reason why `loader.num_workers` was on a a new line: it is a new sentence.

File: docs/source/getting_started/command_line_tool.rst [hunk 2]
```diff
@@ -96,14 +96,14 @@ Lightly-magic is a singular command for training, embedding, and uploading to th
...
 .. code-block:: bash
     
-    # embedding and uploading the images and embeddings to web app
-    # passing in input directory of dataset(Data_dir),using pretrained model
-    # with assigned token as yourToken and datasedID as yourDatasedID
-    # loader.num_workers species the number of cpu cores used for speeding the process.
+    # Embedding and uploading the images and embeddings to web app passing in 
+    # input directory of dataset(Data_dir), using pretrained model with assigned 
+    # token as yourToken and datasedID as yourDatasedID loader.num_workers specifies 
+    # the number of cpu cores used for speeding up the process
```
Comment (MalteEbner) [inline docs/source/getting_started/command_line_tool.rst:102]: Maybe it would be better to rewrite this whole comment in full sentences.

**Review by MalteEbner — COMMENTED** [review]
Great that you did not only solve the issue, but even made additional improvements!
There is only one paragraph I would improve.

File: docs/source/getting_started/command_line_tool.rst (hunk 1 above)
Comment (philippmwirth) [reply docs/source/getting_started/command_line_tool.rst:101]: also, it's dataset 😉

File: docs/source/getting_started/command_line_tool.rst (hunk 2 above)
Comment (philippmwirth) [reply docs/source/getting_started/command_line_tool.rst:102]: dataset

File: docs/source/getting_started/command_line_tool.rst [hunk 3]
```diff
@@ -95,15 +95,16 @@ Lightly-magic is a singular command for training, embedding, and uploading to th
...
 .. code-block:: bash
-    
-    # embedding and uploading the images and embeddings to web app
-    # passing in input directory of dataset(Data_dir),using pretrained model
-    # with assigned token as yourToken and datasedID as yourDatasedID
-    # loader.num_workers species the number of cpu cores used for speeding the process.
+
+    # Embed and upload images from an input directory to an existing dataset in 
+    # the web app. Setting trainer.max_epochs=0 uses a pretrained model for 
+    # embedding and loader.num_workers=8 specifies the number of cpu cores used
```
Comment (MalteEbner) [inline docs/source/getting_started/command_line_tool.rst:101]: " and " -> .
Two sentences instead of one makes it easier to read.

**Review by MalteEbner — APPROVED** [review]
LGTM

**Pull request merged**